content/ja/about/_index.md

```diff
@@ -0,0 +1,123 @@
+        <p style="text-align:left">インナーソースについてこれから学ぶ方、インナーソースが実装しているのが誰なのか、なぜなのかを詳しく知りたい方は、サイトの "学ぶ" セクションにアクセスしてください。トレーニング資料やベストプラクティス、書籍や過去のイベントの記録、ケーススタディなど、インナーソースの採用に役立つリソースが豊富に用意されています。
+        </p>
+        <a href="learn/" class="btn-link">インナーソースについて学ぶ<i class="ti-arrow-right"></i></a>
+      </div>
+      <div class="col-md-6 mt-4 mb-4 mb-md-0 float-right">
+        <img src="/images/community/collaboration.png" class="img-fluid pl-4 pr-4">
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="section">
+  <div class="container">
+    <div class="row align-items-center">
+      <div class="col-md-5 mb-4 mb-md-0">
+        {{< image-community >}}
+      </div>
+      <div class="col-md-6">
+          <p class="section-title h2">コミュニティに参加する</p>
+          <p>インナーソースの実践者やインナーソースを学びたい方を幅広い活動でサポートします。開始するには、Slack を介して InnerSource Commons のコミュニティに参加し、自己紹介をしてください。<br />インナーソースの旅をともにする素晴らしい仲間を見つけることができますよ！</p>
+          <a href="/community/" class="btn-link">コミュニティへの参加<i class="ti-arrow-right"></i></a>
+        </div>
+    </div>
+  </div>
+</section>
+
+<section class="section bg-light">
+  <div class="container">
+    <div class="row align-items-center">
+      <div class="col-md-12">
+        <p class="h2 section-title text-center">InnerSource Commons の動画</p>
+        <div id="youmax" class=""></div>
+        {{< more-videos-button >}}
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="section">
+  <div class="container">
+    <div class="row align-items-center">
+      <div class="offset-md-2 col-md-4 mb-4 mb-md-0">
+        {{< announcements >}}
+      </div>
+      <div class="col-md-5">
+        <p class="section-title h2">InnerSource Commons財団</p>
+        <p>InnerSource Commons は、米501(c)(3)認定をうけた非営利団体として運営され、一連の内規により管理されています。</p>
+        <p><a href="/about/" class="btn-link">財団について<i class="ti-arrow-right"></i></a></p>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="section-last section">
+  <div class="container section-small shadow rounded-lg px-4 bg-light">
+    <div class="row align-items-center justify-content-center text-center text-md-left">
+      <div class="col-lg-5 col-md-4 mb-4 mb-md-0">
+        <a class="twitter-timeline" data-height="500" data-dnt="true" href="https://twitter.com/InnerSourceOrg?ref_src=twsrc%5Etfw"></a> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+      </div>
+      <div class="col-md-5 offset-md-1">
+        <p class="h2 section-title">最新情報はこちら</p>
+        <p class="mb-4">
+        コミュニティ活動のほとんどは、<a href="https://innersourcecommons-inviter.herokuapp.com/">Slack</a> で行われています。また、メールの定期購読や、 <a href="https://twitter.com/InnerSourceOrg">Twitter</a>、 <a href="https://www.linkedin.com/company/innersourcecommons">LinkedIn</a> のフォロー、<a href="https://www.youtube.com/channel/UCoSPSd6Or4F_vpjo4SmyoEA">YouTube</a>チャンネルの登録やによって、InnerSource Commons のニュース、イベントについての情報を取得することができます。</p>
```

Comment:
Since heroku inviter is going to disappear soon, suggest you to change the slack link to https://innersourcecommons.org/slack/